CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+# dbt_reddit_ads v1.3.0
+[PR#31](https://github.com/fivetran/dbt_reddit_ads/pull/31) includes the following updates:
+
+## Schema/Data Change
+**72 total changes • 0 possible breaking changes**
```

Comment:
```suggestion
**12 total changes • 0 possible breaking changes**
```
This maybe seems more appropriate since there are 12 new fields/rows in this table in total